src/config/sentry.ts

```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore file */
```

Comment:
Add comment when ignoring. "We ignore Sentry in tests/coverage because it's a configuration file for our error handler."